### PR TITLE
EOS-17348 : Main - Changed sspl support bundle temporary file location.

### PR DESCRIPTION
**Pull request by satish-seagate**
## Problem Statement
<pre>
  <code>
    Story Ref (if any):
    Please add Problem statement here...
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Please add Problem decription here...
  </code>
</pre>
## Solution
<pre>
  <code>
    Please add summary of the change,List any dependencies that are required for this change.
  </code>
</pre>
## Sanity testing on RPM done
<pre>
  <code>
    - [ ] Yes
    - [ ] No
  </code>
</pre>
## Unit/Manual Testing Description
<pre>
  <code>
    Please describe the tests that you ran to verify your changes.
  </code>
</pre>

### CODE REVIEW

File: low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate [hunk 1]
```diff
@@ -27,11 +27,11 @@ import subprocess
 sys.path.insert(0, f'/opt/seagate/cortx/sspl/low-level')
 from framework.base.sspl_constants import (PRODUCT_FAMILY, SUPPORT_REQUESTOR_NAME, SUPPORT_EMAIL_ID, SUPPORT_CONTACT_NUMBER,
      ENCL_TRIGGER_LOG_MAX_RETRY, ENCL_DOWNLOAD_LOG_MAX_RETRY, ENCL_DOWNLOAD_LOG_WAIT_BEFORE_RETRY, CONSUL_DUMP,
-     sspl_config_path)
+     sspl_config_path, DATA_PATH)
```
Comment (thavanathan) [inline low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate:30]: continuation line under-indented for visual indent

Also please make lines under 80 chars length.

Comment (satish-seagate) [reply low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate]: Done

File: low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate [hunk 2]
```diff
@@ -301,12 +302,12 @@ if __name__ == "__main__":
     if not os.path.isfile(onGoingRequestPath):
         if len(sys.argv) >= 3:
             tmpFile = open(onGoingRequestPath, "w")
-            os.makedirs(f"/tmp/{sspl_bundle_tmp}")
```
Comment (thavanathan) [inline low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate:304]: We can also cleanup the `sspl_bundle_tmp` if it already exists. If any of below three steps fails for some reason, we may have left over files.

Comment (satish-seagate) [reply low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate:304]: Done

**Review by cdeshmukh — APPROVED** [review]

**Review by thavanathan — APPROVED** [review]

**Pull request merged**